### ISSUE TITLE
Can't target 2.2 and 3.0 in single project, fails to restore Microsoft.NETCore.App

### ISSUE THREAD

**Issue by adamsitnik** [dotnet/cli#10074]
## Steps to reproduce

1. Install .NET Core 2.2
2. Install .NET Core 3.0
3. Create new project
4. Target both "netcoreapp2.2" and "netcoreapp3.0"
5. Run dotnet restore

## Expected  behavior

Packages get restored

## Actual behavior

`Unable to find a stable package Microsoft.NETCore.App with version (>= 2.2.0)`

## Environment data

Happens when using latest .NET Core 2.2 and 3.0 bits.

We hit this issue in the perf repo. Is it by design? Are we doing something wrong? We want to compare perf 2.0 vs 2.1 vs 2.2 vs 3.0

/cc jorive

**Comment by livarcocc** [comment]
2.2 is not released yet, so in 3.0, we don't have an implicit version for it set.

You could set RuntimeFrameworkVersion to a particular runtime conditioned on the runtime that you are targeting.

I am not sure we want to keep updating 3.0 without nightlies of 2.2 at this point.

**Comment by adamsitnik** [comment]
@livarcocc big thanks for a very fast reply! your suggestion solved the issue. 

```xml
    <TargetFrameworks>netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
    <RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp2.2'">2.2.0-preview2-26905-02</RuntimeFrameworkVersion>
```
